ws-nextjs-app/pages/[service]/new_live/[id]/Post/index.tsx

```diff
@@ -34,6 +36,37 @@ const unorderedList = ({ blocks }: { blocks: OptimoBlock[] }) => {
   );
 };
 
+const PostBreakingNewsLabel = ({
+  isBreakingNews,
+  breakingNewsLabelText = 'Breaking',
+}: {
+  isBreakingNews: boolean;
+  breakingNewsLabelText?: string;
+}) => {
+  return isBreakingNews ? (
+    <Text css={styles.breakingNewsLabel} data-testid="breaking-news-label">
```

Comment:
Just checking against the designs, does this need to be bold/semi-bold?

Reply:
Thank you for the feedback Aaron.
I'll run this by Wendy for clarification because the text appears
visually bold but the font-style is set to `normal` in the designs
when examined in `Dev Mode`.

Reply:
Thanks Alex. The `<Text>` component can be passed a `fontVariant` prop that sets the `font-weight` and `font-style` CSS properties if needed.

Reply:
I agree with asking UX to clarify. I tend to use the Specs page in Figma over the inspector. 

I interpret this as 'Brevier Bold' having been specified by UX for the label. So we would need to also specify`size="brevier"` alongside the `fontVariant="sansBold"` to achieve this.

![Screenshot 2023-09-18 at 11 50 06](https://github.com/bbc/simorgh/assets/98817636/39a61f80-5da4-4a3a-9da2-7f0d63751afc)

Reply:
Thanks for the pointers, both.
I've confirmed with Wendy that the text should be visually bold.
The visual bold is achieved by the `font-weight` in `sansBold` and
not `font-style`.
This change has been effected.